### PR TITLE
Starmap V and a visit from an old friend!

### PR DESCRIPTION
**Pull request by Bokkiewokkie**
## About The Pull Request

Around the world around the woorld

## Why It's Good For The Game

🌍 🌏 🌎 

## Changelog
:cl:
add: Added the NSV Hammerhead
config: Added Starmap V: Rosetta Cluster
/:cl:

### CODE REVIEW

**Comment by Bobbanz1** [comment]
Map issue here
Screenshot 2022-09-30 223739
Tube ain't connected to ANYTHING

**Comment by Bobbanz1** [comment]
Screenshot 2022-09-30 225456
This wire just leads to fucking space

**Comment by BetaCentari** [comment]
TGUI on the ChemMaster 3000 crashes every ten seconds or so.

**Comment by Karmic-Skink** [comment]
No fun allowed.

**Comment by someone543** [comment]
Hammerhead cargo is missing a freight torpedo launcher

File: nsv13/code/modules/overmap/types/nanotrasen.dm [hunk 1]
```diff
@@ -182,14 +182,24 @@
...
 	starting_system = "Staging" //Required for all player ships
 	armor = list("overmap_light" = 95, "overmap_medium" = 60, "overmap_heavy" = 20)
 	overmap_deletion_traits = DAMAGE_STARTS_COUNTDOWN
+
+/obj/structure/overmap/nanotrasen/heavy_cruiser/starter/apply_weapons()
+	weapon_types[FIRE_MODE_AMS] = new /datum/ship_weapon/vls(src)
+	weapon_types[FIRE_MODE_GAUSS] = new /datum/ship_weapon/gauss(src)
+	weapon_types[FIRE_MODE_PDC] = new /datum/ship_weapon/pdc_mount(src)
+	weapon_types[FIRE_MODE_MISSILE] = new /datum/ship_weapon/missile_launcher(src)
+	weapon_types[FIRE_MODE_RAILGUN] = new /datum/ship_weapon/hybrid_railgun(src)
```
Comment (someone543) [inline nsv13/code/modules/overmap/types/nanotrasen.dm:201]: If we're using hybrid railguns, shouldn't this be FIRE_MODE_HYBRID_RAILGUN?

File: tools/hooks/Install.bat [hunk 2]
```diff
@@ -1,2 +0,0 @@
-@call "%~dp0\..\bootstrap\python" -m hooks.install %*
```
Comment (covertcorvid) [inline tools/hooks/Install.bat:1]: Why was this changed?

File: _maps/aetherwhisp.json [hunk 3]
```diff
@@ -12,5 +12,6 @@
 	    "emergency": "emergency_donut"},
 	"mine_disable": 1,
 	"ship_type": "/obj/structure/overmap/nanotrasen/solgov/aetherwhisp/starter",
-	"faction": "solgov"
+	"faction": "solgov",
+	"starmap_path": "config/starmap/starmap.json"
```
Comment (covertcorvid) [inline _maps/aetherwhisp.json:16]: Is SSmapping actually set up to use this?

**Review by Karmic-Skink — CHANGES_REQUESTED** [review]
As per noticing a bug fix.

Remove the gateway on Hammerhead please thanks.

**Comment by covertcorvid** [comment]
Can we stop fleets from persisting into the starting system? If we don't let them play around after completing their objectives, there's no way for them to ensure Argo's cleared out.

File: _maps/aetherwhisp.json (hunk 3 above)
Comment (Bokkiewokkie) [reply _maps/aetherwhisp.json:16]: Yeah it should be on 182 of map_config.dm

File: tools/hooks/Install.bat (hunk 2 above)
Comment (Bokkiewokkie) [reply tools/hooks/Install.bat:1]: I didn't touch it, must've gone wrong with a master merge somewhere.

**Comment by Bobbanz1** [comment]
Bots get stuck trying to patrol between the theater and the escape shuttle hallway

**Comment by Bobbanz1** [comment]
Three examples of the pathfinding issue they seem to have with those nodes.

Screenshot 2022-10-21 204048
Screenshot 2022-10-21 204035
Screenshot 2022-10-21 210241

**Comment by Bobbanz1** [comment]
Bot got stuck patrolling for a few loops down by the Security Lobby for some unknown reason.

Screenshot 2022-10-28 220746
Screenshot 2022-10-28 220736
Screenshot 2022-10-28 220636

File: nsv13/code/modules/overmap/traders.dm [hunk 4]
```diff
@@ -169,10 +169,10 @@
 	desc = "Ship construction deeds done cheap (for a price)"
 	shortname = "CZC"
 	faction_type = FACTION_ID_NT
-	greetings = list("Welcome to CzanekCorp, we take cash, credit, and charge. What’cha need?",\
+	greetings = list("Welcome to CzanekCorp, we take cash, credit, and charge. What'cha need?",\
 	"CzanekCorp here. We got a new shipment in, you down for talking turkey?",\
 	"CzanekCorp, we got repairs and goods on a budget, you in?")
-	on_purchase = list("Yes, we know the tazers aren’t the safest, but if you don’t like ‘em, stop buying ‘em, eh?", "Good doing business with you. Good luck out there, killer.", "About time we got somebody who knows what they’re doing. Here, free shipping!", "No refunds, no returns!")
+	on_purchase = list("Yes, we know the tazers aren't the safest, but if you don't like 'em, stop buying 'em, eh?", "Good doing business with you. Good luck out there, killer.", "About time we got somebody who knows what they're doing. Here, free shipping!", "No refunds, no returns!")
```
Comment (Karmic-Skink) [inline nsv13/code/modules/overmap/traders.dm:175]: "Tazer"

I'll properly review this later, sorry.

**Review by Karmic-Skink — CHANGES_REQUESTED** [review]

File: nsv13/code/modules/overmap/traders.dm (hunk 4 above)
Comment (Bokkiewokkie) [reply nsv13/code/modules/overmap/traders.dm:175]: They're called that everywhere in the code, we talked about this before and if you want to change it I suggest doing it everywhere at once another time.

File: nsv13/code/modules/overmap/types/nanotrasen.dm [hunk 5]
```diff
@@ -182,14 +182,24 @@
...
 	starting_system = "Staging" //Required for all player ships
 	armor = list("overmap_light" = 95, "overmap_medium" = 60, "overmap_heavy" = 20)
 	overmap_deletion_traits = DAMAGE_STARTS_COUNTDOWN
+
+/obj/structure/overmap/nanotrasen/heavy_cruiser/starter/apply_weapons()
+	weapon_types[FIRE_MODE_AMS] = new /datum/ship_weapon/vls(src)
+	weapon_types[FIRE_MODE_GAUSS] = new /datum/ship_weapon/gauss(src)
+	weapon_types[FIRE_MODE_PDC] = new /datum/ship_weapon/pdc_mount(src)
+	weapon_types[FIRE_MODE_MISSILE] = new /datum/ship_weapon/missile_launcher(src)
+	weapon_types[FIRE_MODE_HYBRID_RAIL] = new /datum/ship_weapon/hybrid_railgun(src)
```
Comment (Karmic-Skink) [inline nsv13/code/modules/overmap/types/nanotrasen.dm:201]: Hammerhead doesn't meet the requirements for using Hybrid Railguns.

**Review by Karmic-Skink — DISMISSED** [review]

File: nsv13/code/modules/overmap/types/nanotrasen.dm (hunk 5 above)
Comment (Bokkiewokkie) [reply nsv13/code/modules/overmap/types/nanotrasen.dm:201]: Will switch to broadside cannons when merged instead 👍
